cli/cmd/plugin/unmanaged-cluster/kapp/kapp.go

```diff
@@ -62,20 +60,19 @@ type KappInstallOpts struct {
 	Manifests [][]byte
 }
 
-// KappManager defines the interface for performing kapp operations.
-//nolint:golint
-type KappManager interface {
+// Manager defines the interface for performing kapp operations.
+type Manager interface {
 	// Install installs kapp-controller into the cluster. When successful, it returns the Deployment object that
 	// manages the kapp-controller pod.
-	Install(opts KappInstallOpts) (*v1.Deployment, error)
+	Install(opts InstallOpts) (*v1.Deployment, error)
 	// Status retrieves the pod status for kapp-controller. It expects to be passed the namespace and name for the
 	// kapp-controller Deployment object. If it cannot talk to the cluster, that status is reported. If the
 	// pod cannot be resolved, a status of not created is reported. Otherwise, the exact status message is returned.
 	Status(ns, name string) string
 }
 
 // New instantiates a new KappManager.
-func New(kubeconfigBytes []byte) (KappManager, error) {
+func New(kubeconfigBytes []byte) (Manager, error) {
```

Comment:
I'm _alittle_ worried about overloading the term `Manager` throughout the codebase here

So far, it seems like we have a `Manager` for the `cluster`, `kapp`, `kubeconfig`, `tanzu`, etc which _may_ be confusing as a developer working in multiple packages of the library.

But i suppose it _does_ make more sense from an API end user perspective to call something as `*.Manager` with it's package name like `kapp.Manager` instead of the more verbose `{package}.{package}Manager`

🦆

Reply:
> But i suppose it does make more sense from an API end user perspective

I hold this perspective. Similar to how go recommends using `New()` in a package (e.g. dog) rather than `dog.NewDog` since the package new effectively the prefix.